scripts/Interface/ui/drill/model/ParameterController.py

```diff
@@ -0,0 +1,161 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import queue
+import threading
+import time
+
+from qtpy.QtCore import QObject, Signal
+
+import mantid.simpleapi as sapi
+
+
+class Parameter:
```

Comment:
Here as well, maybe we should prefix the classes and the file with `Drill` to be consistent with the rest?

Reply:
I renamed it.